<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bernd Stramm &lt;bernd.stramm@gmail.com&gt;</author>
 <class>SubscriptionChange</class>
 <widget class="QDialog" name="SubscriptionChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visibility Request</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../dchat.qrc">
    <normaloff>:/dchatlogo.png</normaloff>:/dchatlogo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>You:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>They:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="changeMessage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>20</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="allowButton">
       <property name="text">
        <string>Allow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="denyButton">
       <property name="text">
        <string>Deny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ignoreButton">
       <property name="text">
        <string>Ignore</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="ownId">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPlainTextEdit" name="remoteId">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources>
  <include location="../dchat.qrc"/>
 </resources>
 <connections/>
</ui>
